src/Compilers/CSharp/Test/Syntax/Parsing/ParserErrorMessageTests.cs

```diff
@@ -4158,26 +4158,34 @@ struct S<out U> { }
     delegate void D<in U>();
     class B<out U> { }
 }";
-
-            ParseAndValidate(test,
-                // (3,12): error CS1960: Invalid variance modifier. Only interface and delegate type parameters can be specified as variant.
-                Diagnostic(ErrorCode.ERR_IllegalVarianceSyntax, "in").WithLocation(3, 12),
-                // (4,12): error CS7002: Unexpected use of a generic name
-                Diagnostic(ErrorCode.ERR_UnexpectedGenericName, "this").WithLocation(4, 12),
```

Comment:
@jcouv Have addressed the test.  The issue was that i was only verify 'declaration diagnostics' not 'all diagnostics'.  Once i included all diagnostics, we see that parse error return.